h2o-algos/src/test/java/hex/grid/GridTest.java

```diff
@@ -108,7 +221,10 @@ public void testFailedParamsRetention() {
       params._train = trainingFrame._key;
       params._response_column = "species";
 
-      Job<Grid> gs = GridSearch.startGridSearch(null, params, hyperParms);
+      GridSearch.SimpleParametersBuilderFactory simpleParametersBuilderFactory = new GridSearch.SimpleParametersBuilderFactory();
+      HyperSpaceSearchCriteria.RandomDiscreteValueSearchCriteria hyperSpaceSearchCriteria = new HyperSpaceSearchCriteria.RandomDiscreteValueSearchCriteria();
```

Comment:
@Pscheidl I did this to kind of fix the test. With `RandomDiscreteValueSearchCriteria` we have default `max_models=0`(means no early stopping) whereas with `CartesianSearchCriteria` we have it set to hyperSpaceSize size by default ( and this would prevent us from evaluating extra grid items)